app/controllers/warehouse_reports/hmis_exports_controller.rb

```diff
@@ -45,6 +45,7 @@ def create
         if frequency.positive?
           recurring_export = GrdaWarehouse::RecurringHmisExport.create(recurrence_params.merge(user_id: current_user.id))
           @filter.recurring_hmis_export_id = recurring_export.id
+          @filter.zip_password = recurring_export.zip_password
```

Comment:
Does the filter also need `encryption_type`?

Reply:
Actually, the opposite, this was accidentally left in from a previous attemp.